dev/com.ibm.websphere.appserver.features/visibility/public/mpRestClient-3.1/io.openliberty.mpRestClient-3.1.feature

```diff
@@ -0,0 +1,34 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpRestClient-3.1
```

Comment:
I think that the MP Rest Client feature version should have been 4.0. If no other major issues are found with this PR then I will change that in a follow-on PR.

Reply:
fixed